src/main/java/seedu/heymatez/model/HeyMatez.java

```diff
@@ -149,6 +149,14 @@ public void removeAssignee(Name name) {
         tasks.removeAssignee(name);
     }
 
+    /**
+     * Replaces the given Assignee {@code targetName} with {@code editedName} in tasks from the task list.
```

Comment:
I think @code refers to the class? Not sure about this

Reply:
@code can refer to the parameter input too